PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,22 @@
-First of all, please read [`CONTRIBUTING.md`](https://github.com/mattermost/desktop/blob/master/CONTRIBUTING.md)
+Thank you for planning to submit a pull request and contributing to the Mattermost Desktop application!
```

Comment:
This message might be left on the pull requests, so possibly this should be written on the top of `CONTRIBUTING.md` instead of here.